astropy/time/core.py

```diff
@@ -990,7 +1136,14 @@ def argmin(self, axis=None, out=None):
         """
         # first get the minimum at normal precision.
         jd = self.jd1 + self.jd2
-        approx = jd.min(axis, keepdims=True)
+
+        if NUMPY_LT_1_11:
+            # MaskedArray.min ignores keepdims so do it by hand
+            approx = np.min(jd, axis)
+            if axis is not None:
+                approx = np.expand_dims(approx, axis)
+        else:
+            approx = np.min(jd, axis, keepdims=True)
```

Comment:
@taldcroft - I think the issue is here for the CircleCI issue. On CircleCI we have Numpy 1.11, and it looks like the last line doesn't actually preserve dimensions:

```
(Pdb) np.__version__
'1.11.0'
(Pdb) np.min(jd, axis, keepdims=True)
masked_array(data =
 [[2450000.6 -- 2450000.6 2450001.4 2450001.4]
 [2450050.6 -- 2450050.6 2450051.4 2450051.4]],
             mask =
 [[False  True False False False]
 [False  True False False False]],
       fill_value = 1e+20)

(Pdb) np.min(jd, axis, keepdims=True).shape
(2, 5)
```

Should ``NUMPY_LT_1_11`` be ``NUMPY_LE_1_11`` or ``NUMPY_LT_1_12``?

Reply:
@mhvk - is this related to the fix that was included in 1.11.2? (https://github.com/numpy/numpy/pull/8018) If so then I guess this should be NUMPY_LT_1_11_2.

Reply:
@astrofrog - ah right.  When I was testing locally I just tried numpy 1.11 which was the latest bugfix version.  Will fix.

Reply:
@astrofrog - nice sleuthing!